### PR TITLE
Taking care of your vvv

### LINKED ISSUE #845: Upgrade VVV
What is the correct procedure to upgrade VVV to its latest edition?

### PR DESCRIPTION
**Pull request by tomjn**
- updates install docs
 - updates soft requirements page
 - add s keeping your VVV up to date ( fixes https://github.com/Varying-Vagrant-Vagrants/VVV/issues/845 )

### CODE REVIEW

File: docs/en-US/installation/index.md [hunk 1]
```diff
@@ -7,15 +7,32 @@ permalink: /docs/en-US/installation/
...
+  * OR download and extract a [stable release](https://github.com/varying-vagrant-vagrants/vvv/releases) zip file if you'd like some extra comfort.
+
+## Starting VVV
+
 1. In a command prompt, change into the new directory with `cd vagrant-local`
 1. Start the Vagrant environment with `vagrant up`
     * Be patient as the magic happens. This could take a while on the first run as your local machine downloads the required files.
     * Watch as the script ends, as an administrator or `su` ***password may be required*** to properly modify the hosts file on your local machine.
-1. Visit any of the [built in WordPress sites](../references/default-sites.md) or [the VVV Dashboard at http://vvv.test](http://vvv.test)
+1. Visit any of [the VVV Dashboard at http://vvv.test](http://vvv.test) to view your new WordPress sites
```
Comment (jeremyfelt) [inline docs/en-US/installation/index.md]: Looks like "any of" can now be removed.

File: docs/en-US/installation/software-requirements.md [hunk 2]
```diff
@@ -25,3 +25,7 @@ VVV requires recent versions of both Vagrant and VirtualBox to be installed, alo
         * If vagrant-triggers is not installed, VVV will not provide automated database backups.
     1. Install the [vagrant-vbguest](https://github.com/dotless-de/vagrant-vbguest) plugin with `vagrant plugin install vagrant-vbguest`.
         * Note: This step is not a requirement. When installed, it keeps the [VirtualBox Guest Additions](https://www.virtualbox.org/manual/ch04.html) kernel modules of your guest synchronized with the version of your host whenever you do `vagrant up`. This can prevent some subtle shared folder errors.
+
+## Windows Users
+
+You may need to turn on virtualisation in your computers BIOS, some computers come with it turned off by default. Refer to your machines manufacturer for how to access your BIOS.
```
Comment (jeremyfelt) [inline docs/en-US/installation/software-requirements.md]: For better or worse, we already use the US English "virtualization", so we should probably stick with that. 😝

File: docs/en-US/installation/keeping-up-to-date.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+category: 2. Getting Started
+order: 3
+title: Keeping Up To Date
+permalink: /docs/en-US/installation/keeping-up-to-date
```
Comment (jeremyfelt) [inline docs/en-US/installation/keeping-up-to-date.md]: Should this have a trailing slash on it for consistency after #32?

**Review by jeremyfelt — DISMISSED** [review]
Looking good, I left a few comments inline.

File: docs/en-US/installation/keeping-up-to-date.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+ 3. Run `vagrant box update` ( optional but probably for the best )
+ 4. `git pull` if you used git
+    1.  if you downloaded instead though, download and extract a new copy of VVV over the top. Make sure that your existing sites in `www` aren't overwritten, and you have DB backups
+    2. If you used `git` you should be fine, as long as you had `vagrant triggers`
+ 5. Now your copy of VVV is up to date
+ 6. Run `vagrant up --provision` to update the box and bring it up
+
+### A Note On Preserving Data
+
+VVV is intended as a developer environment, and you should be able to throw away the VM and rebuild it without loosing anything. Having said that, do not keep critical information stored only in VVV, always take backups.
```
Comment (jeremyfelt) [inline docs/en-US/installation/keeping-up-to-date.md]: s/loosing/losing/

File: docs/en-US/installation/software-requirements.md (hunk 2 above)
Comment (tomjn) [reply docs/en-US/installation/software-requirements.md]: Force of habit, British english man using British english :p

**Comment by tomjn** [comment]
Thanks @jeremyfelt I've fixed up the issues

**Comment by jeremyfelt** [comment]
Nice, thanks @tomjn!

**Review by jeremyfelt — APPROVED** [review]

**Comment by skorasaurus** [comment]
Thank you. \0/ :)

**Pull request merged**